### PR TITLE
Feat/search page

### PR DESCRIPTION
**Pull request by Hannes-HH**
Another short pull request 

- Remove Inline Styling 
- Move Themeprovider to App.js 
- Change Header Fontsize 
- Removed ThemeProvider CSS 

Thanks for reviewing!

### CODE REVIEW

File: src/components/Header/Header.css [hunk 1]
```diff
@@ -7,8 +7,8 @@
   text-transform: uppercase;
   background-color: var(--main-bg-color);
   font-family: "Montserrat", sans-serif;
-  font-size: 3vw;
-  font-weight: 900;
+  font-size: 5.5vw;
+  /* font-weight: 900; */
```
Comment (FriederikeM) [inline src/components/Header/Header.css:11]: you can delete this :)

**Review by FriederikeM — APPROVED** [review]
Hey Johannes, this looks good to me. I really like the design of you app!
Cool that you are using the material framework :)
I left the tiniest comment

File: src/components/Header/Header.css [hunk 2]
```diff
@@ -7,8 +7,8 @@
   text-transform: uppercase;
   background-color: var(--main-bg-color);
   font-family: "Montserrat", sans-serif;
```
Comment (hugh-burgess) [inline src/components/Header/Header.css:9]: you can make this a custom variable if you use the same background colours

File: src/components/Header/Header.css [hunk 3]
```diff
@@ -7,8 +7,8 @@
...
-  font-size: 3vw;
-  font-weight: 900;
+  font-size: 5.5vw;
+  /* font-weight: 900; */
   box-shadow: 0px 1px 5px var(--main-color);
```
Comment (hugh-burgess) [inline src/components/Header/Header.css:12]: this could all be a custom variable too

File: src/components/MainNav.js [hunk 4]
```diff
@@ -42,7 +42,7 @@ export default function SimpleBottomNavigation() {
     >
       <BottomNavigationAction
         style={{ color: "white" }}
```
Comment (hugh-burgess) [inline src/components/MainNav.js:44]: you can make a classname and then implement this style

**Review by hugh-burgess — CHANGES_REQUESTED** [review]
fantastic mate keep it up! couple things.

File: src/components/Header/Header.css (hunk 1 above)
Comment (Hannes-HH) [reply src/components/Header/Header.css:11]: Hey Rikki, many thank for reviewing. Cool that you took the time. I will delete that.

File: src/components/Header/Header.css (hunk 2 above)
Comment (Hannes-HH) [reply src/components/Header/Header.css:9]: Hey Hugh, many thanks for reviewing. Cool that you took the time. I will change this.

File: src/components/Header/Header.css (hunk 3 above)
Comment (Hannes-HH) [reply src/components/Header/Header.css:12]: I will change this.

File: src/components/MainNav.js (hunk 4 above)
Comment (Hannes-HH) [reply src/components/MainNav.js:44]: I will change this.

**Pull request merged**